### PR TITLE
update README

### PR DESCRIPTION
**Pull request by rscohn2**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,33 +51,35 @@ Installing the Intel(R) MLSL using RPM Package Manager (root mode):
...
-    3. Uninstalling the Intel(R) MLSL using the RPM Package Manager
+    3. Uninstalling Intel(R) MLSL using the RPM Package Manager
 
         $ rpm -e intel-mlsl-devel-64-<version>.<update>-<package#>.x86_64
 
-Installing the Intel(R) MLSL using install.sh (user mode):
+Installing Intel(R) MLSL using the tar file (user mode):
 
-    1. Run install.sh and follow the instructions.
+        $ tar txf l_mlsl-devel-64-<version>.<update>.<package#>.tgz
```
Comment (mshiryaev) [inline README.md]: we can't combine "t" and "x" in the same command, maybe just "tar xf <file>" ?

Comment (ykiryano) [reply README.md]: I guess it is a misprint, it should be "zxf"

Comment (rscohn2) [reply README.md]: Right, it should be zxf. I will update the PR

**Comment by rscohn2** [comment]
I didn't squash the 2 commits, but you should be able to squash in the merge.

**Comment by SmorkalovME** [comment]
Robert, thank you for the contribution. This pull request will be handled using corporate requirements and guidelines. It will be pushed in one of upcoming releases, with corresponding notes in Release Notes (What’s New section). As soon the release is out, this pull request will be closed.

**Comment by SmorkalovME** [comment]
Thanks for contribution Robert! PR has been merged.

**Pull request merged**